drivers/ce_performance/app/models/ce_performance/client.rb

```diff
@@ -166,6 +166,68 @@ class Client < GrdaWarehouseBase
       joins(:source_client).merge(GrdaWarehouse::Hud::Client.gender_different_identity)
     end
 
+    scope :valid_exit_destination, -> do
+      joins(:source_client).merge(
+        GrdaWarehouse::Hud::Client.
+        joins(source_enrollments: :exit).
+        distinct.
+        where(ex_t[:Destination].in(HudUtility2024.valid_destinations.keys)),
+      )
+    end
+    scope :in_exit_destination, ->(destination_id) do
+      joins(:source_client).merge(
+        GrdaWarehouse::Hud::Client.
+        joins(source_enrollments: :exit).
+        distinct.
+        where(ex_t[:Destination].in(destination_id)),
+      )
+    end
+
+    scope :homeless_exit_destination, -> do
+      joins(:source_client).merge(
+        GrdaWarehouse::Hud::Client.
+        joins(source_enrollments: :exit).
+        distinct.
+        where(ex_t[:Destination].in(HudUtility2024.homeless_destinations)),
+      )
+    end
+
+    scope :institutional_exit_destination, -> do
+      joins(:source_client).merge(
+        GrdaWarehouse::Hud::Client.
+        joins(source_enrollments: :exit).
+        distinct.
+        where(ex_t[:Destination].in(HudUtility2024.institutional_destinations)),
+      )
+    end
+
+    scope :temporary_exit_destination, -> do
+      joins(:source_client).merge(
+        GrdaWarehouse::Hud::Client.
+        joins(source_enrollments: :exit).
+        distinct.
+        where(ex_t[:Destination].in(HudUtility2024.temporary_destinations)),
+      )
+    end
+
+    scope :permanent_exit_destination, -> do
+      joins(:source_client).merge(
+        GrdaWarehouse::Hud::Client.
+        joins(source_enrollments: :exit).
+        distinct.
+        where(ex_t[:Destination].in(HudUtility2024.permanent_destinations)),
+      )
+    end
+
+    scope :other_exit_destination, -> do
+      joins(:source_client).merge(
+        GrdaWarehouse::Hud::Client.
+        joins(source_enrollments: :exit).
+        distinct.
+        where(ex_t[:Destination].in(HudUtility2024.other_destinations)),
+      )
+    end
```

Comment:
Since you added the `in_exit_destination` scope, seems like these should all use it.

```suggestion
    scope :homeless_exit_destination, -> do
      in_exit_destination(HudUtility2024.homeless_destinations)
    end

    scope :institutional_exit_destination, -> do
      in_exit_destination(HudUtility2024.institutional_destinations)
    end

    scope :temporary_exit_destination, -> do
      in_exit_destination(HudUtility2024.temporary_destinations)
    end

    scope :permanent_exit_destination, -> do
      in_exit_destination(HudUtility2024.permanent_destinations)
    end

    scope :other_exit_destination, -> do
      in_exit_destination(HudUtility2024.other_destinations)
    end
```

Reply:
makes sense, the in_exit_destination scope was added later in the changes, but it would 100% work to reduce the redundant code here. I'll include it in the next commit.